tests/poly_ops.mlir

```diff
@@ -1,25 +1,28 @@
-// RUN: heir-opt %s | FileCheck %s
+// RUN: heir-opt %s > %t
+// RUN: FileCheck %s < %t
 
 // This simply tests for syntax.
 
+#my_poly = #poly.polynomial<1 + x**1024>
+#my_poly_2 = #poly.polynomial<t**3 + t + 2>
+#my_poly_3 = #poly.polynomial<2>
+#my_poly_4 = #poly.polynomial<x>
```

Comment:
were these needed?

Reply:
I intended to have them to handle all the parser edge cases, but now I'm realizing I didn't add any integer coefficients to the terms with nonzero degree, so I am updating this one slightly to have that.